### DISCUSSION TITLE
Help in writing a script that simply voices the srt file.

### DISCUSSION THREAD

**Discussion by bropines** [coqui-ai/TTS#3084] [General Q&A]
Let's say I'm extremely stupid and I need help. I wanted to use your algorithm to voice the subtitle file. And at first the problem appeared in the fact that I stupidly do not understand which model is suitable. XTTS seems to work, but it seems like she voices two words and, for fun, takes part of the recording from the original track. Then I would like to have just a high-quality tts engine to simply voice and run through the SVC to match my voice.

Can anyone help with studying which tts is best suited here, as well as with writing an algorithm. I would ask GPT yes he is dumb as a cork and even giving him full documentation he is dumb as hell.

**Comment by marioguima** [comment]
@bropines
Regarding which TTS model is better, I have the same question.
About the str file, I used open ai whisper
https://github.com/openai/whisper

**Reply by bropines** [reply]
Whisper makes text from audio. And I need to voice the SRT file. I’ll write this creak with a lot of tears, but I tried xttsv2 and even though it’s extremely robotic
